### DISCUSSION TITLE
install and use cloud technology

### DISCUSSION THREAD

**Discussion by HILALOZTEMEL** [mlcommons/GaNDLF#462] [Ideas]
Is there a way to install and use gandlf in google colab or other cloud technology?

**Comment by Geeks-Sid** [comment]
Hi,

We have not explicitly tested on cloud-based servers, but you could use the [dockers](https://www.docker.com/) provided to run GaNDLF on any platform that supports docker.

**Comment by AlexanderGetka-cbica** [comment]
Hi @HILALOZTEMEL , while we don't have anything on Google Colab right now, it is something we are looking at for the very near future. The current plan is to use it as a tutorial for GaNDLF but it should be a good starting point.

**Comment by sarthakpati** [comment]
We have had an intern who used GaNDLF with Google colab. You will need to install it using the git link directly.

**Reply by HILALOZTEMEL** [reply]
how can i get the link?
